scripts/tests/test_zkai_native_seq32_attention_mlp_expanded_label_probe_gate.py

```
@@ -0,0 +1,157 @@
+            tsv_path = tmp_path / "out.tsv"
+            self.gate.write_outputs(json_path, tsv_path, self.payload)
+            loaded = json.loads(json_path.read_text())
+            self.assertEqual(loaded["payload_commitment"], self.payload["payload_commitment"])
+            lines = tsv_path.read_text().splitlines()
+            self.assertEqual(len(lines), 1 + len(self.payload["proof_object_rows"]))
+            self.assertIn("payload_commitment", lines[0])
+
+    def test_rejects_output_paths_outside_evidence_dir(self):
+        with tempfile.TemporaryDirectory() as tmp:
+            tmp_path = pathlib.Path(tmp)
+            with self.assertRaisesRegex(
```

Ruff (0.15.13)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)